changes/796.feature.md

```diff
@@ -0,0 +1,3 @@
+Added a total_length() function to hikari.Embed
```

Comment:
```suggestion
Added a `total_length` function to `hikari.embeds.Embed`
```

Reply:
This seems a tad bit overly pedantic and i don't think it should block a pr lol